test/webhook-proxy.test.ts

```diff
@@ -18,14 +24,14 @@ describe('webhook-proxy', () => {
     beforeEach((done) => {
       app = express()
 
-      app.get('/events', sse, function (req, res) {
-        res.json({}, 'ready')
+      app.get('/events', sse, (req, res) => {
+        res.json({}, 'ready') // fixme: not sure how to convert this to TypeScript
         emit = res.json
       })
 
       server = app.listen(0, () => {
-        url = `http://127.0.0.1:${server.address().port}/events`
-        proxy = createWebhookProxy({url, port: targetPort, path: '/test', logger})
+        const url = `http://127.0.0.1:${server.address().port}/events`
```

Comment:
Oops, forgot to look before opening the PR, but I'm seeing an error message on this line, will look into addressing it:

```
Property 'port' does not exist on type 'string | AddressInfo'.
  Property 'port' does not exist on type 'string'.
```